cmdupload/upload.go

```diff
@@ -263,7 +274,8 @@ func NewUpCmd(ctx context.Context, ic *immich.ImmichClient, logger *logger.Logge
 	cmd.BoolVar(&app.KeepTrashed, "keep-trashed", false, "Import also trashed items")
 	cmd.BoolVar(&app.KeepPartner, "keep-partner", true, "Import also partner's items")
 	cmd.BoolVar(&app.CreateAlbumAfterFolder, "create-album-folder", false, "Create albums for assets based on the parent folder or a given name")
-	cmd.StringVar(&app.ImportIntoAlbum, "album", "", "All assets will be added to this album.")
+	cmd.StringVar(&app.ImportIntoAlbum, "album", "", "All assets will be added to this album. (Must already exist)")
```

Comment:
```suggestion
	Must already exist
```
Not true. Albums are created on the fly id needed

Reply:
I didn't track down the root cause, but some albums fail to get created. This happened multiple times and I actually ended up changing upload.go to remove the `return` from line 440 ("can't create the album list from server") so it prints the error, but doesn't stop creating additional albums. The error on the server side when an album fails is a constraint violation on table albums_assets_assets so I'm guessing maybe the list of assets to be added to the album needs to be de-duped.

Reply:
When an album fails to get created the server throws a 500 error, but the message to the client doesn't have any details.

Reply:
I'm trying to replicate your situation. How you end up with duplicates in the albums? I suspect that assets are present in several albums. Is that true?

Reply:
yes, 84k assets into ~60 albums. Most assets were in 3+ albums